src/pages/patientView/PatientViewPage.tsx

```diff
@@ -342,8 +346,22 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         let cohortNav: JSX.Element | null = null;
         let studyName: JSX.Element | null = null;
 
+        if (patientViewPageStore.urlValidationError) {
+            return (
+                <div className="alert alert-danger urlError" role="alert">
+                    <i className="fa fa-warning" aria-hidden="true"></i>
+                    <h3>The URL is invalid</h3>
+                    <ul>
+                        { patientViewPageStore.urlValidationError
+                            .split(".").map((message:string)=>(message.length > 0) ? <li>{message}</li> : null)
+                        }
+                    </ul>
+                </div>
+            )
+        }
+
         if (patientViewPageStore.studyMetaData.isComplete) {
-            let study: CancerStudy = patientViewPageStore.studyMetaData.result;
+            let study: CancerStudy = (patientViewPageStore.studyMetaData.result as CancerStudy);
```

Comment:
I think if you remove the default value for `studyMetaData` over in `PatientViewPageStore`, you don't need to cast it because if `studyMetaData.isComplete` then there's a valid `CancerStudy`